src/localization/locales/en-US.json

```diff
@@ -95,7 +110,16 @@
         "list_column_location": "Location",
         "list_column_website": "Website",
         "list_column_members": "Members",
-        "list_empty": "No Landscapes"
+        "list_empty": "No Landscapes",
+        "members_title_manager": "Manage Members",
+        "members_description_manager": "Remove or invite members for your {{name}} to maintain the accurate member list in {{name}} profile page.",
+        "members_title_member": "{{name}} Members",
+        "members_description_member": "",
+        "members_list_leave": "Leave",
+        "members_list_remove": "Remove",
+        "members_remove_confirmation_title": "Remove Member",
+        "members_remove_confirmation_message": "You are about to remove “{{userName}}” from the “{{name}}” member list.",
```

Comment:
issue: same as above with group. Make it clear you are removing them from the landscape.

Reply:
@paulschreiber Same as above, please check with Hana and let me know the improved content to add here

Reply:
Please change to `"You are about to remove “{{userName}}” from the “{{name}}” landscape."`